src/type-generator.ts

```diff
@@ -33,6 +34,11 @@ export interface TypeGeneratorOptions {
    * @default true
    */
   readonly toJson?: boolean;
+
+  /**
+   * Normalization function to apply to references which are used to create
+   */
+  readonly normalizeRef?: (s: string) => string;
```

Comment:
Not in love with this. But I thought we would need this in `cdk8s` so we don't end up with horrible looking interface names.

Reply:
I think I would call this `renderTypeName(def: string) => string`

Reply:
Changed